docs/remote/containers.md

```diff
@@ -1067,86 +450,6 @@ See the [Advanced Container Configuration](/docs/remote/containers-advanced.md)
 * [Developing inside a container on a remote Docker Machine or SSH host](/docs/remote/containers-advanced.md#developing-inside-a-container-on-a-remote-docker-host)
 * [Reducing Dockerfile build warnings](/docs/remote/containers-advanced.md#reducing-dockerfile-build-warnings)
 
-## devcontainer.json reference
```

Comment:
This is a common jump point - so I wonder if we should keep the heading so it shows up on the right nav but then just point to the dedicated topic. This is also one I'd expect to see people directly linking to - unless we can meta redirect.

For example, it is referenced in the GitHub docs: https://docs.github.com/en/github/developing-online-with-codespaces/configuring-codespaces-for-your-project

Reply:
Yep, agreed. I haven't been able to sort out a redirect and I found a link in the docs.microsoft.com Codespaces docs as well.  I also think more visibility (right nav) will be helpful. I'm reconsidering having the devcontainer.json reference be hidden and probably good to have it visible in the TOC.